site/_includes/primary-nav-items.html

```diff
@@ -10,6 +10,6 @@
     <li class="training-menu"><a href="/news/">News</a>
     </li>
     <li>
-        <button class="button-default button-green trigger-top-form"> <a href="https://github.com/apache/incubator-metron/releases" target="new">Download </a></button>
+        <button class="button-default button-green trigger-top-form"> <a href="https://dist.apache.org/repos/dist/release/incubator/metron/" target="new">Download </a></button>
```

Comment:
It looks like METRON-600 is now the correct place to state the need for a Downloads page to support the mirroring requirement.  I will add it there, so suggest you go ahead and close METRON-603.